### PR TITLE
Updating upgrade steps

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Documentation around the steps needed to upgrade from 7.5.X or 7.6.X to latest required some further details/highlighting/linking. 

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is clear and complete
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links
* [x] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [x] You have applied a production label if this needs a manual push to Zoomin
* [x] You have added it to the correct GitHub project
* [x] You have added it to the correct Sprint milestone
* [x] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [x] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [x] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md [hunk 1]
```diff
@@ -82,23 +99,25 @@ Check your old installation and identify if you are using any of the following:
 
 Perform the following in your new API Gateway 7.7 installation.
 
-#### Apply the latest service pack
+#### 1. Apply the latest service pack
+
+Apply the latest available service pack for your new installation. To see the system property changes made between the old installation and the latest service pack, please see [System property change list](/docs/apim_installation/apigw_upgrade/upgrade_system_props/). Service packs are available from [Axway Support](https://support.axway.com/).
```
Comment (alexearnshaw) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md:104]: @Bcolemanaxway I'm reviewing this now and this addition has a link to a new file /docs/apim_installation/apigw_upgrade/upgrade_system_props/ but this file is not in the branch/PR. Can you add it please?

Comment (Bcolemanaxway) [reply content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md:104]: Hi Alex, the file actually isn't to do with my ticket. Its being worked on currently as part of another ticket (https://jira.axway.com/browse/RDAPI-20971). I've commented on that ticket and talked to Dave to ensure he uses the file name I've referenced above, so once he adds it we should be okay. Not ideal but wasn't sure how else to approach two separate docs tickets that are linked.

Comment (alexearnshaw) [reply content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md:104]: Ah ok, I'll add a blank file for now then so as not to have a 404.

**Comment by alexearnshaw** [comment]
@Bcolemanaxway Thanks for the improvements in this PR. I didn't keep them all exactly as they were as some were not in line with Axway style or best practices, but I did make some other similar improvements (such as numbering all checklist tasks and steps) to help users identify them easier. Merging this now but feel free to improve further in a future PR.

**Pull request merged**